### PR TITLE
AC automated test LRAC-10248

### PR DESCRIPTION
**Pull request by joshitagaki**
https://issues.liferay.com/browse/LRAC-10248
Bug ticket: https://issues.liferay.com/browse/LRAC-10247
https://github.com/joshitagaki/liferay-portal/pull/38

### CODE REVIEW

**Comment by joshitagaki** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase [hunk 1]
```diff
@@ -271,6 +271,64 @@ definition {
 		ACSegments.viewDeletedSegmentName(searchTerm = "Static Segment Test");
 	}
 
+	@description = "Feature ID: Bug | Automation ID: LRAC-10247 | Test Summary: Deleting a static segment does not remove user in known individuals"
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase:274]: Hey @joshitagaki, I think the description property would be better this way so we know which bug the tests are based on:

`@description = "Bug: LRAC-10247 | Automation ID: LRAC-10247 | Test Summary: Deleting a static segment does not remove user in known individuals"`

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase [hunk 2]
```diff
@@ -271,6 +271,64 @@ definition {
...
+	@priority = "5"
+	test CheckIfKnownIndividualsDontDisappearAfterDeleteStaticSegment {
+		var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase:277]: Hey @joshitagaki, you don't need to put a variable to receive the name of the property, the connection macro should no longer return the name of the property, inside the getAssignedPropertyName macro the variable is being declared as global, so we can use its value directly in the tests (I believe it was missing remove the return from the macro, I had made this change in the tests that were in the old and updated way, but I think there are still some. I'll open a ticket to adjust this, but could you fix that file if possible?):

**Before:**
```
var propertyName = ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");


ACProperties.switchProperty(propertyName = "${propertyName}");
```

**Now:**
```
ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");


ACProperties.switchProperty(propertyName = "${assignedPropertyName}");
```

**Review by ViniLopes20 — CHANGES_REQUESTED** [review]
Hey @joshitagaki, I will put more suggestions for you:

- Put tasks to explain what you are doing in some parts of the code

- I think that after verifying that the individual is in the list of known individuals you could put the test to create the segment again with that individual and in this way assure that it appears in the list to be added by the segment again after the deletion of the previous one

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase (hunk 2 above)
Comment (joshitagaki) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase:277]: Oh nice! I'll make those changes

**Comment by joshitagaki** [comment]
Thanks for the suggestions @ViniLopes20!

**Comment by joshitagaki** [comment]
ci:test:sf

**Comment by joshitagaki** [comment]
ci:test:sf

**Review by ViniLopes20 — APPROVED** [review]

**Comment by alexcandidos** [comment]
Starting to review! 👨🏽‍💻

**Comment by alexcandidos** [comment]
Thanks for the suggestions @ViniLopes20 !

Hi @joshitagaki , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**